### PR TITLE
Adding CRUD Tests To User Table

### PR DESCRIPTION
**Pull request by assaftal391**
This is the first tests for User table. It built on top of CRUD model (Create, Read, Update & Delete) which are the basics operations for a table in SQL. If this approach is fine, I'll move one to our other tables.
Note: I changed the configuration for testing to run in-memory tables for efficency considerations.

You can run the tests by using: python -m pytest 

Thanks.

### CODE REVIEW

**Comment by assaftal391** [comment]
Updated the PR as @YossiHananya requested. Main changes:
1. Separating DB testing from the application context (can be tested independently).
2. Use the same session of DB in-memory creation for all CRUD test functions.

File: tests/test_db.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import pytest
+# from .conftest import app
```
Comment (YossiHananya) [inline tests/test_db.py:2]: Remove line

File: tests/test_db.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import pytest
+# from .conftest import app 
+from .conftest import db_session
+from soccer_application.models import User
+from soccer_application import db
```
Comment (YossiHananya) [inline tests/test_db.py:5]: You don't need this import 
Please remove

**Review by YossiHananya — APPROVED** [review]

File: tests/test_db.py (hunk 1 above)
Comment (assaftal391) [reply tests/test_db.py]: fixed

File: tests/test_db.py (hunk 2 above)
Comment (assaftal391) [reply tests/test_db.py]: fixed

**Review by HananyaS — APPROVED** [review]

**Pull request merged**